docs/about/release-notes.md

```diff
@@ -1,5 +1,35 @@
 # Release Notes
 
+
+## ShapeWorks 6.2.1 - 2022-01-07
+
+### What is new?
+
+#### Studio
+  * Fixed display names in the corner (removing long paths)
+  * Added convert to mesh pipeline for segmentation inputs (can run both image and mesh pipelines)
+  * Fixed #1612 and #1616, bugs regarding recomputing of shape statistics
+  * Added reflection (based on spreadsheet column)
+  * Added remeshing tool for meshes (using ACVD library)
+  * Added image cropping to groom
+  * Added image resampling to groom
+  * New UI for grooming page with collapsible panels
+#### New commands / python
```

Comment:
same here, this can be one major frond end feature, e.g. "New commands and Python APIs for meshes"